backend/worker/quivr_worker/celery_worker.py

```diff
@@ -170,18 +167,50 @@ def check_is_premium_task():
     check_is_premium(supabase_client)
 
 
+@celery.task(name="process_sync_task")
+def process_sync_task(
+    sync_id: int, user_id: str, files_ids: list[str], folder_ids: list[str]
+):
+    global async_engine
```

Comment:
ouch

Reply:
This is usual in celery workers. THe state in initiated at the start of the worker. See the init_worker event function